### PR TITLE
hard code steve, alex & skinsrestorer skins

### PR DESCRIPTION
**Pull request by xknat**
fixes https://github.com/SkinsRestorer/SkinsRestorerX/issues/613

### CODE REVIEW

File: src/main/java/net/skinsrestorer/shared/utils/connections/MojangAPI.java [hunk 1]
```diff
@@ -87,16 +90,20 @@ public IProperty createProperty(String name, String value, String signature) {
      * @throws SkinRequestException on not premium or error
      */
     public IProperty getSkin(String nameOrUuid) throws SkinRequestException {
-        IProperty skin = getProfile(nameOrUuid, false);
+        final String finalNameOrUuid = nameOrUuid.trim().toUpperCase();
+        if (Arrays.stream(HardcodedSkins.values()).anyMatch(t -> t.name().equals(finalNameOrUuid))) {
+            System.out.println("MATCH!");
```
Comment (AlexProgrammerDE) [inline src/main/java/net/skinsrestorer/shared/utils/connections/MojangAPI.java:95]: May wanna remove this.

**Review by AlexProgrammerDE — CHANGES_REQUESTED** [review]
lgtm but that one thing has to be removed.

File: src/main/java/net/skinsrestorer/shared/utils/connections/MojangAPI.java (hunk 1 above)
Comment (xknat) [reply src/main/java/net/skinsrestorer/shared/utils/connections/MojangAPI.java:95]: ah yes

Comment (xknat) [reply src/main/java/net/skinsrestorer/shared/utils/connections/MojangAPI.java]: resolved

**Pull request merged**